File: FoodDelivery_Backend/Models/SupplierWallet.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace FoodDelivery_Backend.Models
{
    public class SupplierWallet
    {
        public decimal supplier_wallet_id { get; set; }
        public decimal supplier_id { get; set; }
        public decimal wallet_balance { get; set; }
        public string last_update_date { get; set; }
    }
}

File: FoodDelivery_Angular_Frontend/Frontend-App/src/app/Models/supplierWallet.ts
import { supplierInfoModel } from "./supplierInfo";

export class supplierWalletModel {
    supplier_wallet_id: number;
    supplier_id: number;
    wallet_balance: number;
    supplier_name: string;
    SI:supplierInfoModel;
    last_update_date: string;
    
    constructor() {
        this.supplier_wallet_id = 0;
        this.supplier_id = 0;
        this.wallet_balance=0;
        this.supplier_name = '';
        this.SI=new supplierInfoModel();
        this.last_update_date = '';
    }
}
